acd-config-editor/scripts/readme.md

```diff
@@ -0,0 +1,45 @@
+# Watson Annotator for Clinical Data Configuration Editor stand-alone installation Instructions
```

Comment:
remove this file  -its out of date and for internal how we built the old .tar file for the eap program now anything for clients.